sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/implementation/AzureCommunicationChatServiceImplBuilder.java

```diff
@@ -154,7 +154,7 @@ public AzureCommunicationChatServiceImplBuilder addPolicy(HttpPipelinePolicy cus
      */
     public AzureCommunicationChatServiceImpl buildClient() {
         if (apiVersion == null) {
-            this.apiVersion = "2021-04-05-preview6";
+            this.apiVersion = "2021-09-07";
```

Comment:
Is this class auto-generated? Else we can use `ChatServiceVersion` to avoid updating this manually after every service version update.

Reply:
Yes. This class is auto-generated.